content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -283,7 +298,7 @@ In practice, these are the steps necessary to introspect an existing DB and gene
 
 ```prisma
```

Comment:
This whole code block can probably be indented to be in the list under `1.` instead of breaking one level to the left.